compute_sdk/globus_compute_sdk/serialize/concretes.py

```diff
@@ -310,6 +310,86 @@ def deserialize(self, payload: str, variation: int = 0) -> t.Any:
         raise DeserializationError(f"Deserialization failed after {count} tries")
 
 
+class PureSourceTextInspect(SerializationStrategy):
+    """
+    Extracts a function's definition via :func:`inspect.getsource` and sends the
+    resulting string directly over the wire.
+
+    .. list-table:: Supports
+        :header-rows: 1
+        :align: center
+
+        * - Functions defined in Python interpreter
+          - Code from notebooks
+          - Interop between mismatched Python versions
+          - Decorated functions
+        * - ❌
+          - ✅
+          - ⚠️
+          - ❌
+    """
+
+    identifier = "st\n"
+    for_code = True
```

Comment:
The numbers for identifiers are reasonably "just arbitrary."  Does `st` have any human semantic meaning, if undocumented?

Reply:
"Source textinspect" (and similarly, "source dill" for the other strategy). Thought it might be nice to start using identifiers that are readable at a glance, but if you think we should stick to numeric IDs I'm totally cool with that

Reply:
No, I think this is fine; it just wasn't clear to me for what it was short.  Consider a comment.  Perhaps:

```python
identifier = "st\n"  # "Source Textinspect"
```

That would clear it up for me.  Sometimes it's just catching others up with your thought-process!

Reply:
Added those comments (and also rebased on main).